pkg/apis/controlplane/types.go

```diff
@@ -340,6 +340,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats
+	// Multicast group information related to multicast collected from the Node.
```

Comment:
I think we should remove "related to multicast"

Reply:
updated